src/app/globals.css

```
@@ -1,68 +1,131 @@
-		--border: #e2e8f0;
-		--input: #44444442;
-		--ring: var(--primary);
-		--radius: 0.5rem;
-		--chart-1: #e74c3c;
-		--chart-2: #16a085;
-		--chart-3: #2c3e50;
-		--chart-4: #f1c40f;
-		--chart-5: #e67e22;
+@custom-variant dark (&:where(.dark, .dark *));
+
+@theme {
```

Stylelint (17.2.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)